src/chainparams.cpp

```diff
@@ -101,6 +101,9 @@ class CMainParams : public CChainParams {
             Consensus::NetworkUpgrade::NO_ACTIVATION_HEIGHT;
         consensus.vUpgrades[Consensus::UPGRADE_OVERWINTER].nActivationHeight =
             Consensus::NetworkUpgrade::NO_ACTIVATION_HEIGHT;
+        consensus.vUpgrades[Consensus::BASE_SPROUT].nProtocolVersion = 170002;
+        consensus.vUpgrades[Consensus::UPGRADE_TESTDUMMY].nProtocolVersion = 170002;
+        consensus.vUpgrades[Consensus::UPGRADE_OVERWINTER].nProtocolVersion = 170004;
```

Comment:
Non-blocking: I'd prefer to intersperse these, and match the ordering in `params.h`.

Reply:
ok doing it now

Reply:
Hmm, these are duplicated now.